tests/utils.go

```diff
@@ -2084,12 +2084,15 @@ func GetGuestAgentUserData() string {
 	ipv6UserDataString := ""
 	if netutils.IsIPv6String(dnsServerIP) {
 		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd10:0:2::2/120 dev eth0
+                             sleep 2
```

Comment:
Any option not to make our slow tests even slower with sleeps? Plus it may cause (keep) flakiness on veeery slow systems with emulation enabled.

Reply:
It should take a few msec, I took 2sec as a precaution.
Compared to hundreds of sec it takes this test, this is negligible in terms of time.
But if you have a better solution, please share.

I was thinking of using NM instead of the `ip` command, but that may fit a different change.

Reply:
An alternative to sleep would be polling or monitoring.

>  Compared to hundreds of sec it takes this test, this is negligible in terms of time.

Is not this command executed in virtually every single test on IPv6 lane? Also, this is a dangerous precedent.

Reply:
Ok then, we will retry. Lets see if it works.